### PR TITLE
Issue 262 pitch offset

### PR DESCRIPTION
**Pull request by adler-j**
Solves #262

### CODE REVIEW

File: odl/tomo/geometry/conebeam.py [hunk 1]
```diff
@@ -50,7 +50,8 @@ class HelicalConeFlatGeometry(DivergentBeamGeometry, AxisOrientedGeometry):
     detector positions.
     """
 
-    def __init__(self, angle_intvl, dparams, src_radius, det_radius, pitch,
+    def __init__(self, angle_intvl, dparams, src_radius, det_radius,
+                 pitch, pitch_offset=0,
                  agrid=None, dgrid=None, axis=[0, 0, 1], src_to_det=None,
                  detector_axes=None):
```
Comment (kohr-h) [inline odl/tomo/geometry/conebeam.py]: Make `pitch_offset` and `detector_axes` part of `kwargs`, too many arguments here (even w/o the intervals)

File: odl/tomo/geometry/conebeam.py [hunk 2]
```diff
@@ -155,7 +169,8 @@ def det_refpoint(self, angle):
         circle_component = self.rotation_matrix(angle).dot(origin_to_det)
 
         # Increment by pitch
-        pitch_component = self.axis * self.pitch * angle / (np.pi * 2)
+        pitch_component = self.axis * (self.pitch_offset +
+                                       self.pitch * angle / (np.pi * 2))
```
Comment (kohr-h) [inline odl/tomo/geometry/conebeam.py]: So we have to do that in all places, right? Makes it more complicated, but ok

Comment (adler-j) [inline odl/tomo/geometry/conebeam.py]: Its basically in these two places, right?

File: odl/tomo/geometry/conebeam.py (hunk 1 above)
Comment (adler-j) [inline odl/tomo/geometry/conebeam.py]: Good suggestion

File: odl/tomo/geometry/conebeam.py (hunk 2 above)
Comment (kohr-h) [inline odl/tomo/geometry/conebeam.py]: I don't know. Probably. But also possibly more to come. Anyway, it's a sensible addition.

Comment (adler-j) [inline odl/tomo/geometry/conebeam.py]: Honestly cant see where else, the geometries are uniquely defined by these two points, any utility  would likely just call them and get it for free?

Comment (kohr-h) [inline odl/tomo/geometry/conebeam.py]: Yes, I guess so.

**Comment by adler-j** [comment]
Will fix all these commits, travis has changed something and now it fails.

**Comment by kohr-h** [comment]
Can you clean up with `git rebase -i` when you're done here? Would be good not to have this commit mess in the final git log.

**Comment by adler-j** [comment]
Rebase done, merging.

**Pull request merged**